sound/soc/intel/boards/sof_board_helpers.h

```diff
@@ -44,6 +45,7 @@ struct sof_card_private {
 	int hdmi_num;
 
 	int ssp_codec;
+	int ssp_amp;
```

Comment:
Hmm, where will `ctx->ssp_amp` be used?

Reply:
I planned to submit another PR later which implements a helper function to generate entire DAI link array according to the content of sof_card_private structure. You could check the code here:

https://github.com/brentlu/linux/commit/e259c8a080faf04e9410178ecc5a75828969226c